src/components/header/index.tsx

```diff
@@ -29,10 +43,17 @@ const Header: React.FC = () => {
           <RingIcon />
         </IconsContainer>
         <StyledHr />
-        <StyledTypography variant='body1'>Jones Ferdinand</StyledTypography>
-        <PhotoContainer>
-          <ProfileIcon />
-        </PhotoContainer>
+        <StyledTypography>{`${currentUser.firstName} ${currentUser.lastName}`}</StyledTypography>
```

Comment:
Lets move it to the separate component as well. `userDropdownMenu`

Reply:
fixL userDropdown menu as a sepatrate component